### ISSUE TITLE
Unable to disable Keybase Chat

### ISSUE THREAD

**Issue by vcanfield** [keybase/keybase-issues#2808]
Hi,

For those of us that do not and will not be using the Keybase desktop application, it is frustrating that there does not seem to be a way to disable the ability for users to send us Keybase Chat messages. I have already received one encrypted message through this system, and I will never be able to read it.

There are obvious security concerns with storing these messages, and I would prefer not to have them stockpiled on Keybase's servers. While I think Keybase Chat is a great idea for those that use your desktop application, it is not harmonious with the decentralized, third-party-verifiable principles that I joined Keybase for, and would prefer to disable the feature entirely.

**Comment by CameronBanga** [comment]
As a +1 on this, maybe a chat whitelist? I imagine well known people on Twitter, GitHub, Reddit, etc, are going to be inundated with messages from random people. This may exist today, but I couldn't figure out how.

**Comment by cjb** [comment]
Thanks, CCing @malgorithms on the feature request.

**Comment by vcanfield** [comment]
After almost 3 months, I continue to get encrypted messages that are hoarded on Keybase's servers without my consent. @malgorithms will we get any type of response on this issue?

**Comment by awnumar** [comment]
I agree with this. I don't want to install any of the applications that allow for receiving encrypted messages because they are extremely immature at this time, and so I would appreciate not receiving emails about messages I won't read.

**Comment by malgorithms** [comment]
> I would appreciate not receiving emails about messages I won't read.

The unsubscribe link in the email lets you turn these emails off. I understand this doesn't solve the original request in this thread, but part of yours at least.

**Comment by drusepth** [comment]
Adding a +1 for the original request, having similar concerns.
